platform-sdk/docs/PlatformThreads.drawio

```diff
@@ -0,0 +1 @@
Q6icflxzAe3yBCXZsC/MkA3GJFrlcjVMbSWwymX6/t8Ildj2AvVT5sQAtkFmmSQIt0NkZCNnBBucCa+8zF9hwqroc6Qj4q7J5x/6YN+souniW5L7Z3/Su4Zp4rsm/mvnW23tIApi/0m8TD/IAtBy7Snh8HvhH0s9TBm41uf+erGiPR7vgX+iNQ0v+7u76uukQ0iBAqgPoBpU8bdRTFVz0L/kRTfptIZPfiJc/IZ3SPt/9wnRKJqum6eU1ztuFNtZ6VA97irCTZdDnw7bOoW+vRA54gW2xansmEUdut1aeL2qW/0JTxGuLzg/gMBuDAUBbWY5+AZJ3Pn20LHQa+jw4WziUiu2dNm4pim/wCprH7H6i4kiP6Zb1IW3ehk6bCVwTQ7xm7DmlXggAZtD/sIrwMSBaC6M/skgSdm1IubTwwRY2U7seaqwvRpHrfEdeKAp8EgYA3HSCRjm1ph5NgMBCSA4ojJK8aNCC0YrY/aP2ytveToOdK0b12/dluuK8oVbjUvH17Kkhds75ZGYMlHrYPJpsfKAKPZs7lYgn1QCClq04tnsIXIUyky4V9twnO7Li7/eLBclJts950UkyrRIfXyg5RIFjBbIIcKBKlJ2K/TYgABms1mSBx1muFogjobBX6ubgmjxwhADPCREs7JYMwyFMsO2yfX26lAMzow6xg/ZSnBbGQb9/kvtTPsRputqpTXjEoQbUFmqQeDlwhIn8ePSRBpL3DXbxen0oshwnIAN6/iHgw4sxgTIapAzSCEJelArSUqoazB/GfbXg/GCYzOzANF8DoBCV2Raw62FzwArQWrpWFE8W+GCE7XIQcAsZdCpZTUxQOLzYLT+r8fuJs6DPeQ+1c/49hmhuBCVcJI5pWh0zqAlaz/BtnieeM6vnvGqxZaa3I9aI+kkS3u/mYRZ9iLZREtLZXGPMslXZiF+DD/mI6mGc4CAFqmadIKaSqe1I0Grgo66Q/CL6spp7dCGXBoH6Tmw/7rFUplb/GPTmHiXA9yUn1tkjIqFlGfrmPCJwpS/e4tpcyV23vMG+8A2zPvpN0RspN2YApud8J9F5XkhBl1vDQmMHFg57N+k+vSEA/lLqCTp1yLVb4Hayu1NoZNWJWG7j/Hh/c0//lmWU0BLmMP/vJs7/FxdVbXL8RUwxO+bytzGtSuYWv7jUHVNUUR2vuO2YcsbkN+EcV+KsVIyGqyDrkW1MbnL9YfP5Lb+TpdpNbZMEYCEfh9Cy66UBjmLTysfKgBqIlaEaBSEXWYy2vx84EPGt5HzYElg+qCcsihkyQ2Qs2BnSodHon693QvioUzM7f+ZER6nKdM53dICN+mh3dFh7GUZDLO0LZVazsuZYdyLhOMKYK3x8tIz0Bj4tMBTms6PEph7mM6vXwZGBgLGGmieUQVH3+CeYLACVSbJczO/m8+h78wEjFWQH1n/zAfvEMZTqmqnU9c1kVVj73Fy1FUNbGFGGEB1RVv0T+ghN1+1iuZA/Kk17fEFlXrPbHFQqDlwcJIpkWdwoaBHbiB0aYjd6C+WQQLzOxPxZWyQnctktfK14P/tihht6oIQDfOXwpPY8nSZkOSeeA3r7R1n7uflgLl7cfrr+5en5z9cssP8V/LV5nr/MHDmWyw8GXRAY/yXcaD408vQwzljUhjGwRKbc9WSHUZ4GUvcWgQZORy0OuQh2xcTxc0RcwPicMPa9soK4XEis8Lx0goLC84qtSRrdrpyOPq3DNB3rQrqB5fjVVCl2VgJbVAeNYVFd2TNIK0nngey6r8jpm2cAB1xbQcAvlefTw6+kroP6utFJu4rBleYGUrsdWPM8i/4Ypp+vkyjMCBYOyZq3iL9drYddbpkbP2wuCav3ZveOZkJi6XiCTWDJKPrdRnIHPTK1D27CTE7ESNY6D340lhjJfq+CHD60r5lwx11Fpml4FJSKalQBEFTYl0FHPSj3x1eHH+ccJjcnSm1uXYeLMEMYeLaIsMVdhpdNTIR4vwv7ICLY9IwaauQD7m7X0WZiXUXMIDUNBRaGfmvaVRIYVMg/wOCmkiszzx+XUb421EqhHS/oF6EGVgw2tNNlvF/THx6j/C9frMkaUpraXm3z+w41REn8kuY3ZS9RRF/Y5A7nW6ojarJ1nhnvNfzzZMP+Xv0lBBWu0E8DsYXlyiNTYftv+7/u/dvfwz/89cPm083j/u5HPJOlJmWxv8ZpVpLFD69pFm0+JuE2DZ9o+0RxWu6hJGBKRWEIiCxXgC07qPE1cpcFUm75+sAKoCH5qNgZUT5J9DbFI4Ac9M+VjynjkSmgTrZyZNHfWCkFKktvyijTpYMApB3eOkGvwj8ADNqYTC0dEtlxDrZCMICWUEc8Q62mYfgEA+XCyl0iXbDqWzYP6Aj2MQLzkHcUD+YxzsJkpXnQgm/Hq9mHot57gBM+wJiGyZqZ/d2Vp1GMjaflUkrup1FL2ouOoeKEj2NKb7pOBZX9PmWlAKZj66vDaLp1MupK28tW/ImX/AKtQ1/puXkCoFxpYWeEp4pTIczRM8AftLPv0qGvAYl1LqKFz4WzC6DHU5yneywc7R966ClYuGqL8rpBz6k8NRJ7okERrTYR1dIbZsdnZpMIAB25/2JiWym09smc17m1hZlcgtUDjHaFTQYLmoZyCgOc9HDaQERNE4e5K4UsU9qHSPiBivWBCCXM9klUy2jq6Y4k9sEjZU8sRQW+Y7n1MLlwc/X0Q8WwmMwhm3aZm9e8TsFMYdFtkvJ5R6ubTC56Aw5MJjGZXHRwzOe3xgZmtpplF+oJBXm8yvrYcl1YRQrdAgX5U0ieY9XQRsD1lnCkYUCMtg1MIKN9I+ZMqXq+tG0952GUmRbrlNPi40UCBwtPMJgZsVaJ8UOftjqbI1yL1jqbY0xLK/T11MHXAkot9EzXqVUtAAsAsgCqYZ8wMc80C+TIO1tvA/vKPigd2AeQ4wpy0wJ+4kAzgdg/B/vIr0kcZ/XbSbS7/DmeUynd/hc=</diagram></mxfile>
```

Comment:
Maybe if we store the diagrams as editable SVGs (or editable PNGs), we can view them in github when reviewing a PR

Reply:
Unfortunately with such large diagrams, I fear the github interface will not be much help. I switched the file to svg, though, since I don't see any downsides (everything looks the same, and editing appears to be unaffected)

Reply:
you can still click "raw" and view the file in more detail if you need to, its better i think